database/procedures/generate_dashboard_summary.sql

```diff
@@ -120,15 +120,12 @@ BEGIN
     SET @SQLText = '';
     SET @SQLText = CONCAT(@SQLText, ' INSERT INTO dashboard_project (dashboard_id, project_id)');
     SET @SQLText = CONCAT(@SQLText, ' SELECT ', dashboardYoursId, ', p.id');
-    SET @SQLText = CONCAT(@SQLText, ' FROM projects p');
-    SET @SQLText = CONCAT(@SQLText, ' LEFT JOIN portfolios po');
     SET @SQLText = CONCAT(@SQLText, ' ON p.portfolio_id = po.id');
```

Comment:
Does removing the FROM line here definitely still work ok?

Reply:
> Does removing the FROM line here definitely still work ok?

Oh... Sorry... Just revised...